README.md

```diff
@@ -302,61 +302,45 @@ sudo pacman -S qt5-base \
     qtkeychain-qt5
 ```
 
-##### Gentoo Linux
-
+#### Debian 11+ / Ubuntu 22.04 
+<font size= “1”>*Older OS versions require a newer version of Qt5 than offered in official repositories.*</font>  
```

Comment:
Is this font tag necessary?

Reply:
No, sorry, I was trying to make it smaller, but it doesn't appear to do anything anyway.